variants/esp32-evb/pins_arduino.h

```diff
@@ -29,5 +29,6 @@ static const uint8_t MISO  = 15;
 static const uint8_t SCK   = 14;
 
 #define BOARD_HAS_1BIT_SDMMC
+#define ETH_CAN_INTERFERE_WITH_SDMMC
```

Comment:
Perhaps defining something like `BOARD_MAX_SDMMC_FREQ` would be a bit more generic? Then in sdmmc library you can include pins_arduino.h, check if BOARD_MAX_SDMMC_FREQ isn't defined, and define it to the default value. That would eliminate the ifdef around the begin function.

Reply:
Thanks for idea. Will try it right now.

Reply:
@igrr Sorry for disturb, i do not know how to correct declare about changes, so just calling you in this PR.